spec/index.md

```diff
@@ -5011,6 +5082,8 @@ S with
     reserved_balance_limits[Canister_id] = New_reserved_balance_limit
     certified_data[Canister_id] = ""
     canister_history[Canister_id] = New_canister_history
+    canister_log_visibility[Canister_id] = Controllers
```

Comment:
Here we should take the log visibility from the canister settings if provided.

Reply:
Done.